prov/opx/include/rdma/opx/fi_opx_hfi1_sdma.h

```diff
@@ -212,14 +212,14 @@ void fi_opx_hfi1_dput_sdma_init(struct fi_opx_ep *opx_ep,
 		return;
 	}
 
-	params->delivery_completion = (length >= opx_ep->tx->dcomp_threshold) ||
-				      (is_hmem) ||
-				      (params->opcode == FI_OPX_HFI_DPUT_OPCODE_GET) ||
-				      (params->opcode == FI_OPX_HFI_DPUT_OPCODE_PUT) ||
-				      (params->opcode == FI_OPX_HFI_DPUT_OPCODE_ATOMIC_FETCH) ||
-				      (params->opcode == FI_OPX_HFI_DPUT_OPCODE_ATOMIC_COMPARE_FETCH);
+	params->sdma_no_bounce_buf = (length >= opx_ep->tx->sdma_bounce_buf_threshold) ||
+                            	(is_hmem) ||
+                            	(params->opcode == FI_OPX_HFI_DPUT_OPCODE_GET) ||
+                            	(params->opcode == FI_OPX_HFI_DPUT_OPCODE_PUT) ||
+                            	(params->opcode == FI_OPX_HFI_DPUT_OPCODE_ATOMIC_FETCH) ||
+                            	(params->opcode == FI_OPX_HFI_DPUT_OPCODE_ATOMIC_COMPARE_FETCH);
```

Comment:
Looks line some tabs are accidentally converted to spaces here. Please use tabs as long as possible.